packages/lambda-tiler/src/__test__/tile.import.test.ts

```diff
@@ -120,7 +120,7 @@ o.spec('Import', () => {
   o('should return 200 with existing import', async () => {
```

Comment:
would be good to have a test for the file count and file size limits

Reply:
All fixed. Can you please have a look? Thanks.